src/components/Sidebar/Header.js

```diff
@@ -12,11 +12,12 @@ export const Header = ({
   videoHashId,
   onCloseClick = null,
   imgNewTab = DEFAULT_IMG_NEW_TAB,
-  imgClose = DEFAULT_IMG_CLOSE
+  imgClose = DEFAULT_IMG_CLOSE,
+  param = ""
```

Comment:
`param` is not a very explicit name regarding what it does. Maybe `urlParams` would be a better naming